### DISCUSSION TITLE
How to update docs

### DISCUSSION THREAD

**Discussion by rgstephens** [NVIDIA-NeMo/Guardrails#516] [General]
Is it possible to submit updates to `docs.nvidia.com/nemo/guardrails`?

**Comment by drazvan** [comment]
Hi @rgstephens! The official documentation on [docs.nvidia.com/nemo/guardrails](https://docs.nvidia.com/nemo/guardrails) is generated from the source code in the `/docs` folder. The updates are typically published when new releases are published. So, yes, it is possible. Just create a PR that updates the docs in the `docs` folder.

**Comment by rgstephens** [comment]
I can't find the source for the Colang 2.0 [Defining Flows](https://docs.nvidia.com/nemo/guardrails/colang_2/language_reference/defining-flows.html) page.

**Reply by drazvan** [reply]
Yes, those are yet to be added. The docs for Colang 2.0 have been developed in an internal repo and yet to be added to the public repo. Should show up in the next few weeks.
